templates/mod/security.conf.erb

```diff
@@ -68,13 +68,4 @@
     SecUploadDir /var/lib/mod_security
 <% end -%>
     SecUploadKeepFiles Off
-
-    # ModSecurity Core Rules Set configuration
-<%- if scope.function_versioncmp([scope.lookupvar('::apache::apache_version'), '2.4']) >= 0 -%>
-    IncludeOptional <%= @modsec_dir %>/*.conf
-    IncludeOptional <%= @modsec_dir %>/activated_rules/*.conf
-<%- else -%>
-    Include <%= @modsec_dir %>/*.conf
-    Include <%= @modsec_dir %>/activated_rules/*.conf
-<%- end -%>
```

Comment:
Removing these rules would cause all manually-configured virtualhosts (using `apache::vhost::custom` or normal file resources managing virtualhosts config files) to lose their security rule sets, so we can't take it out.